ansible/roles/test/files/ptftests/copp_tests.py

```diff
@@ -275,7 +275,25 @@ def contruct_packet(self, port_number):
                        eth_src=src_mac,
                        eth_type=0x88cc
                  )
+        return packet
+
+# SONIC configuration has no policer limiting for UDLD
+class UDLDTest(NoPolicyTest):
+    def __init__(self):
+        NoPolicyTest.__init__(self)
+
+    def runTest(self):
+        self.log("UDLDTest")
+        self.run_suite()
 
+    def contruct_packet(self, port_number):
+        src_mac = self.my_mac[port_number]
+        packet = simple_eth_packet(
+                       pktlen=95,
+                       eth_dst='01:00:0c:cc:cc:cc',
+                       eth_src=src_mac,
+                       eth_type=0x0067
```

Comment:
Is this real eth_type or we don't need this at all?

Reply:
this is not the real eth_type. since UDLD is using IEEE 802.3, this is the length of the frame.

Reply:
i'd better leave some comments here 💬

Reply:
if IEEE 802.3 is not supported, can you add a comment here in the code for the eth_type meaning?

Reply:
okay, i have added the comments. please check.